.github/workflows/main.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
```

Comment:
Can we include all the versions of ubuntu and macos. You think we should do docker for centos and debian?

Reply:
Added all supported OS by Github Actions
For Centos and Debian -- let's create a different workflow file